tests/Feature/ReviewTableViewTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
I would rename the test file to something more general like 'RelationalSearchTest'

Reply:
+1 This is a good suggestion.